packages/dev/core/src/FlowGraph/flowGraphVariableDefinitions.ts

```diff
@@ -13,11 +13,24 @@ export class FlowGraphVariableDefinitions {
         this._definitions.set(name, defaultValue);
     }
 
-    public generateContext(params: IFlowGraphGraphVariables): FlowGraphContext {
+    public generateContext(params: IFlowGraphContextConfiguration): FlowGraphContext {
         const context = new FlowGraphContext(params);
         this._definitions.forEach((value, key) => {
             context.setVariable(key, value);
         });
         return context;
     }
+
+    public serialize(serializationObject: any) {
+        this._definitions.forEach((value, key) => {
+            serializationObject[key] = value;
+        });
+    }
+
+    public parse(serializationObject: any) {

```

Comment:
was this missed, or kept like this deliberatly?

Reply:
It was a miss, sorry. I'll change it.